solr/core/src/test/org/apache/solr/update/processor/UUIDUpdateProcessorFallbackTest.java

```diff
@@ -155,17 +155,17 @@ SolrInputDocument doc(SolrInputField... fields) {
   }
 
   /** Convenience method for building up SolrInputFields */
-  SolrInputField field(String name, float boost, Object... values) {
+  SolrInputField field(String name, Object... values) {
     SolrInputField f = new SolrInputField(name);
     for (Object v : values) {
       f.addValue(v);
     }
     return f;
   }
 
-  /** Convenience method for building up SolrInputFields with default boost */
+  /** Convenience method for building up SolrInputFields */
   SolrInputField f(String name, Object... values) {
-    return field(name, 1.0F, values);
+    return field(name, values);
   }
```

Comment:
So now the `f` method has the same signature as the `field` method, and since `f` only calls `field` the `f` method itself can be removed i.e. callers can directly call `field` instead.

Reply:
looks like it's a convenience method, `doc(f("name", "Existing", "Values"), f("id", "75765"))`

Reply:
So, I removed the `field` method since `f` is used everywhere....      

I suspect that this pattern is spread across lots of tests, and it seems like moving some methods into some unit testing helpers might be good...